File: server/ee91dd2e-f7d0-4893-a2ca-45cccaab49fc/function.go
package main



import (

    "os"

)



func checkCeltAssociation() {

    celt1 := os.Getenv("MY_VARIABLE_INPUT_1") // קבל את הערך של CELT1 מהמשתנים הסביבתיים

    celt2 := os.Getenv("MY_VARIABLE_OUTPUT") // קבל את הערך של CELT2 מהמשתנים הסביבתיים



    if celt1 == celt2 {

         os.Exit(0) // מקושר

    } else {

         os.Exit(1) // לא מקושר

    }

}



func main() {

    checkCeltAssociation()

}

File: server/7206672b-ecf5-4bf7-ba5f-fa5b05a8db84/function.js
  function checkCeltAssociation() {

        var celt1 = process.env.MY_VARIABLE_INPUT_1; // Retrieve the value of CELT1 from environment variables

        var celt2 = process.env.MY_VARIABLE_OUTPUT; // Retrieve the value of CELT2 from environment variables

        console.log("celt1",celt1)

        console.log("celt2",celt2)



        if (celt1 === celt2) {

            process.exit(0)

        } else {

            process.exit(1)

        }

    }

    

    console.log(checkCeltAssociation());